examples/collection_capabilities.ipynb

```diff
@@ -0,0 +1,159 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Harmony Py Library\n",
+    "### Collection Capabilities Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import helper\n",
+    "import json\n",
+    "import sys\n",
+    "sys.path.append('..')\n",
+    "\n",
+    "# Install harmony-py requirements.  Not necessary if you ran `pip install harmony-py` in your kernel  \n",
+    "!{sys.executable} -m pip install -q -r ../requirements/core.txt\n",
+    "\n",
+    "from harmony import BBox, Client, Collection, Request, Environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Get collection capabilities"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "harmony_client = Client(env=Environment.UAT)\n",
+    "\n",
+    "collection = Collection(id='C1234088182-EEDTEST')\n",
```

Comment:
We should add another cell that shows you can also specify the collection to use by providing a collection short name instead of a CMR concept ID.

Reply:
I am not even aware that we support ShortName as a parameter on capabilities endpoint. This is why we should have a more detailed API doc in Harmony. Filed ticket: harmony-1525.